attributes/default.rb

```diff
@@ -29,8 +29,13 @@
 default['nrpe']['package']['options'] = nil
 
 # nrpe daemon user/group
-default['nrpe']['user']  = 'nagios'
-default['nrpe']['group'] = 'nagios'
+if node['platform_version'].to_i == 7
```

Comment:
You'll want to make sure the platform_family is RHEL and version 7 otherwise you'll catch Debian here too